modules/ipi-install-preparing-the-provisioner-node-for-openshift-install.adoc

```diff
@@ -149,6 +149,13 @@ The IPv6 address can be any address as long as it is not routable via the `barem
 Ensure that UEFI is enabled and UEFI PXE settings are set to the IPv6 protocol when using IPv6 addressing.
 ====
 
+. Configure IPv4 address on provisioning connection.
```

Comment:
"provisioning connection" is confusing to me. If the meaning holds, I suggest this:
```suggestion
. Configure the IPv4 address on the `provisioning` network connection.
```

Reply:
acked